rj_gameplay/rj_gameplay/tactic/seek.py

```diff
@@ -13,20 +14,21 @@ def __init__(
         self,
         world_state: stp.rc.WorldState,
         num_seekers: int,
-        formations: List,
-        centroids: List,
+        formations: stp.formations.Formations,
     ):
         super().__init__(world_state)
 
-        formation = formations
-        centroid_list = centroids
+        formation = formations.get_regions
```

Comment:
These names are confusing. The class being passed in is "DiamondFormation" but this param is named formations; the local var formation is then assigned to the param formations.get_regions?

It seems like you should change `formations` to `formation`, then change this var to `region = formation.get_regions` to avoid confusion.